src/components/Navigator/NavigatorCard.vue

```diff
@@ -337,8 +352,14 @@ export default {
         // check if `title` matches the pattern, if provided
         const titleMatch = filterPattern ? filterPattern.test(title) : true;
         // check if `type` matches any of the selected tags
-        const tagMatch = selectedTags.length
-          ? tagsSet.has(TOPIC_TYPE_TO_TAG[type]) : true;
+        let tagMatch = true;
+        if (selectedTags.length) {
+          tagMatch = tagsSet.has(TOPIC_TYPE_TO_TAG[type]);
+          // if there are API changes and there is no tag match, try to match change types
+          if (apiChanges && !tagMatch) {
```

Comment:
Why do we only try to match change types when there are API Changes and no tag match? 
I was thinking maybe we should always match both Topic and API Changes? So we can further filter the content down.